projects/plugins/crm/webpack.config.js

```diff
@@ -186,4 +187,58 @@ module.exports = [
 			} ),
 		],
 	},
+	{
+		entry: {
+			'onboarding-wizard': './src/js/onboarding-wizard-index.jsx',
+		},
+		mode: jetpackWebpackConfig.mode,
+		devtool: jetpackWebpackConfig.isDevelopment ? 'source-map' : false,
+		output: {
+			...jetpackWebpackConfig.output,
+			path: path.resolve( './build' ),
+		},
```

Comment:
We're going to make a slow transition towards a full React UI which means that we won't have a single index main file which the original intent of this setup (copied from the starter plugin) is meant for.

We can still define different builds in `entry` and the key will be used as the filename, but all of our components would be living in a flat directory structure with the current config. Perhaps we should try and create a directory per build instead?

Reply:
Yes, we could define different outputs. The Jetpack plugin itself has done similar, an example being in `webpack.config.extensions.js`. An [entry point](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tools/webpack.config.extensions.js#L175) includes [all blocks in the extensions folder with `view.js` files](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tools/webpack.config.extensions.js#L52) as I understand it, and the output is [a folder for each block](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tools/webpack.config.extensions.js#L104) under `_inc/blocks`.

Reply:
Great suggestion 👏 I've borrowed the concept and implemented it in a simpler way than the Jetpack plugin.
We still look for `view.{js,jsx,ts,tsx}` files amongst our components in a way that makes it Jetpack Assets loader compatible.

Example:
`projects/plugins/crm/src/js/components/onboarding-wizard/view.jsx`
=>
`projects/plugins/crm/build/onboarding-wizard/index.asset.php`
`projects/plugins/crm/build/onboarding-wizard/index.css`
`projects/plugins/crm/build/onboarding-wizard/index.css.map`
`projects/plugins/crm/build/onboarding-wizard/index.js`
`projects/plugins/crm/build/onboarding-wizard/index.js.map`
`projects/plugins/crm/build/onboarding-wizard/index.rtl.css`

Reply:
Nice!